src/lib/operators/export_binary.hpp

```diff
@@ -73,7 +73,7 @@ class ExportBinary : public AbstractReadOnlyOperator {
    * Row count             | ChunkOffset                           |  4
    *
    * Next, it dumps the contents of the segments in the respective format (depending on the type
-   * of the segment, such as ReferenceSegment, DictionarySegment, ValueSegment).
+   * of the segment, such as ReferenceSegment, DictionarySegment, ValueSegment, RunLengthSegment).
```

Comment:
I wouldn't expect the "such as" list to be complete - otherwise it would be also missing the FixedStringDictionarySegment.